### ISSUE TITLE
I tried using a tool like T2V to train a Wan LoRA model，real woman, but all my attempts failed. Do you have any suggestions?

### ISSUE THREAD

**Issue by ShanQueTit** [kohya-ss/musubi-tuner#170]
I tried to use a bust or mugshot as a training dataset, but the face and limbs became distorted after 6 rounds of training

I haven't seen good t2v character training in Civitai now. Do you have any successful experiences to share?

**Comment by ShanQueTit** [comment]
dit = "./ckpts/wan/split_files/diffusion_models/wan2.1_t2v_14B_bf16.safetensors
vae = "./ckpts/vae/Wan2.1_VAE.pth"                                                 
t5 = "./ckpts/text_encoder/models_t5_umt5-xxl-enc-bf16.pth"                           
clip = "./ckpts/text_encoder_2/models_clip_open-clip-xlm-roberta-large-vit-huge-14.pth"
$attn_mode = "flash"                                                           
$mixed_precision = "bf16
--fp8_base

**Comment by Sarania** [comment]
What was your LR? I haven't tried to train any character LoRA in Wan yet but I think it likes a lower LR than Hunyuan. Also if things go too many steps it tends to get wonky. I've had good luck with LR=5.e5, LoraPlus=4, steps = 2400

**Comment by ShanQueTit** [comment]
> What was your LR? I haven't tried to train any character LoRA in Wan yet but I think it likes a lower LR than Hunyuan. Also if things go too many steps it tends to get wonky. I've had good luck with LR=5.e5, LoraPlus=4, steps = 2400 for an object/motion LoRA.您的 LR 是什么？我还没有尝试在 Wan 中训练任何角色 LoRA，但我认为它喜欢比 Hunyuan 更低的 LR。此外，如果事情进展太多，它往往会变得不稳定。我很幸运，物体/运动 LoRA 的 LR=5.e5、LoraPlus=4、步长 = 2400。

t2v ILR=2e-4，i used 5e-4 It will make the picture worse and faster.
I2v, I have tried a few, 2e-4, lora work very well.
Can you try to train t2v character lora and share some experience with me?

**Comment by Sarania** [comment]
> > What was your LR? I haven't tried to train any character LoRA in Wan yet but I think it likes a lower LR than Hunyuan. Also if things go too many steps it tends to get wonky. I've had good luck with LR=5.e5, LoraPlus=4, steps = 2400 for an object/motion LoRA.您的 LR 是什么？我还没有尝试在 Wan 中训练任何角色 LoRA，但我认为它喜欢比 Hunyuan 更低的 LR。此外，如果事情进展太多，它往往会变得不稳定。我很幸运，物体/运动 LoRA 的 LR=5.e5、LoraPlus=4、步长 = 2400。
> 
> t2v , LR=2e-4，i used 5e-4 It will make the lora worse. I2v, I have tried a few, 2e-4, lora work very well. Can you try to train t2v character lora and share some experience with me?

I personally think 2e-4 is too much, I don't even use that high with Hunyuan. What you're talking about with faces and limbs becoming distorted happens when the model's gradients start to collapse from overtraining(e.g. too many steps, too high LR).

I will get around to training a character at some point and let you know, but my poor 4070 Ti Super doesn't ever rest lol. I do notice there is a stark lack of character LoRA for Wan I don't know if that's harder or just because I2V is so good or what. Anyway for now this is my most successful Wan training config:

Date: 2025-03-14T05:59:03 Title: -------
Resolution: 1280x720 Architecture: wan2.1/lora
Network Dim/Rank: 16.0 Alpha: 1.0  dtype: BF16
Module: networks.lora_wan : {'loraplus_lr_ratio': '4'}
Learning Rate (LR): 6e-05
Optimizer: came_pytorch.CAME.CAME
Scheduler: constant_with_warmup  Warmup steps: 100
Epoch: 56 Batches per epoch: 40 Gradient accumulation steps: 1
Timestep Sampling: Shift Discrete Flow Shft: 3.0

But my experience training Wan is limited and I'm certain can do better.

**Comment by ShanQueTit** [comment]
> > > What was your LR? I haven't tried to train any character LoRA in Wan yet but I think it likes a lower LR than Hunyuan. Also if things go too many steps it tends to get wonky. I've had good luck with LR=5.e5, LoraPlus=4, steps = 2400 for an object/motion LoRA.您的 LR 是什么？我还没有尝试在 Wan 中训练任何角色 LoRA，但我认为它喜欢比 Hunyuan 更低的 LR。此外，如果事情进展太多，它往往会变得不稳定。我很幸运，物体/运动 LoRA 的 LR=5.e5、LoraPlus=4、步长 = 2400。您的 LR 是什么？我还没有尝试在 Wan 中训练任何角色 LoRA，但我认为它喜欢比 Hunyuan 更低的 LR。此外，如果事情进展太多，它往往会变得不稳定。我运气很好，LR=5.e5， LoraPlus=4， steps = 2400 for an object/motion LoRA.您的 LR 是什么？我还没有尝试在 Wan 中训练任何角色 LoRA，但我认为它喜欢比 Hunyuan 更低的 LR。此外，如果事情进展太多，它往往会变得不稳定。我很幸运，物体/运动 LoRA 的 LR=5.e5、LoraPlus=4、步长 = 2400。
> > 
> > 
> > t2v , LR=2e-4，i used 5e-4 It will make the lora worse. I2v, I have tried a few, 2e-4, lora work very well. Can you try to train t2v character lora and share some experience with me?t2v，LR=2e-4，我用了 5e-4 它会让 lora 变得更糟。I2v，我试过一些，2e-4，lora 效果很好。你能试着训练 t2v 角色 lora 并与我分享一些经验吗？
> 
> I personally think 2e-4 is too much, I don't even use that high with Hunyuan. What you're talking about with faces and limbs becoming distorted happens when the model's gradients start to collapse from overtraining(e.g. too many steps, too high LR).我个人觉得 2e-4 太多了，我什至没有对 Huunyuan 使用那么高。你说的面部和四肢变得扭曲发生在模型的梯度开始因过度训练而崩溃时（例如，步数太多，LR 太高）。
> 
> I will get around to training a character at some point and let you know, but my poor 4070 Ti Super doesn't ever rest lol. I do notice there is a stark lack of character LoRA for Wan I don't know if that's harder or just because I2V is so good or what. Anyway for now this is my most successful Wan training config:我会在某个时候抽身训练一个角色并让你知道，但我可怜的 4070 Ti Super 永远不会休息，哈哈。我确实注意到 Wan 明显缺乏角色 LoRA，我不知道这是更难还是仅仅因为 I2V 太好了或其他什么。无论如何，目前这是我最成功的 Wan 训练配置：
> 
> Date: 2025-03-14T05:59:03 Title: -------日期：2025-03-14T05：59：03 标题：------- Resolution: 424x240(actual training res) Architecture: wan2.1/lora分辨率：424x240（实际训练分辨率） 架构：wan2.1/lora Network Dim/Rank: 16.0 Alpha: 1.0 dtype: BF16网络亮度/等级：16.0 Alpha：1.0 dtype：BF16 Module: networks.lora_wan : {'loraplus_lr_ratio': '4'}模块： networks.lora_wan ： {'loraplus_lr_ratio'： '4'} Learning Rate (LR): 6e-05学习速率 （LR）：6e-05 Optimizer: came_pytorch.CAME.CAME优化器：came_pytorch。 来。 来 Scheduler: constant_with_warmup Warmup steps: 100计划程序：constant_with_warmup 预热步骤：100 Epoch: 56 Batches per epoch: 40 Gradient accumulation steps: 1Epoch：56 个批次 每个 epoch：40 梯度累积步骤：1 Timestep Sampling: Shift Discrete Flow Shft: 3.0时间步采样：Shift Discrete Flow Shft：3.0
> 
> But my experience training Wan is limited and I'm certain can do better.但我训练 Wan 的经验是有限的，我确信可以做得更好。

I often see your experience sharing in the AI forum. I am very grateful to you. I use the poorer 4060ti16g, but we can all enjoy the fun of AI. i look forward to your work.

**Comment by ShanQueTit** [comment]
> > > > What was your LR? I haven't tried to train any character LoRA in Wan yet but I think it likes a lower LR than Hunyuan. Also if things go too many steps it tends to get wonky. I've had good luck with LR=5.e5, LoraPlus=4, steps = 2400 for an object/motion LoRA.您的 LR 是什么？我还没有尝试在 Wan 中训练任何角色 LoRA，但我认为它喜欢比 Hunyuan 更低的 LR。此外，如果事情进展太多，它往往会变得不稳定。我很幸运，物体/运动 LoRA 的 LR=5.e5、LoraPlus=4、步长 = 2400。您的 LR 是什么？我还没有尝试在 Wan 中训练任何角色 LoRA，但我认为它喜欢比 Hunyuan 更低的 LR。此外，如果事情进展太多，它往往会变得不稳定。我运气很好，LR=5.e5， LoraPlus=4， steps = 2400 for an object/motion LoRA.您的 LR 是什么？我还没有尝试在 Wan 中训练任何角色 LoRA，但我认为它喜欢比 Hunyuan 更低的 LR。此外，如果事情进展太多，它往往会变得不稳定。我很幸运，物体/运动 LoRA 的 LR=5.e5、LoraPlus=4、步长 = 2400。
> > > 
> > > 
> > > t2v , LR=2e-4，i used 5e-4 It will make the lora worse. I2v, I have tried a few, 2e-4, lora work very well. Can you try to train t2v character lora and share some experience with me?t2v，LR=2e-4，我用了 5e-4 它会让 lora 变得更糟。I2v，我试过一些，2e-4，lora 效果很好。你能试着训练 t2v 角色 lora 并与我分享一些经验吗？t2v ， LR=2e-4，我用了 5e-4 它会让 lora 变得更糟。I2v，我试过一些，2e-4，lora 效果很好。你能试着训练 t2v 角色 lora 并与我分享一些经验吗？t2v，LR=2e-4，我用了 5e-4 它会让 lora 变得更糟。I2v，我试过一些，2e-4，lora 效果很好。你能试着训练 t2v 角色 lora 并与我分享一些经验吗？
> > 
> > 
> > I personally think 2e-4 is too much, I don't even use that high with Hunyuan. What you're talking about with faces and limbs becoming distorted happens when the model's gradients start to collapse from overtraining(e.g. too many steps, too high LR).我个人觉得 2e-4 太多了，我什至没有对 Huunyuan 使用那么高。你说的面部和四肢变得扭曲发生在模型的梯度开始因过度训练而崩溃时（例如，步数太多，LR 太高）。我个人觉得 2e-4 太多了，我什至没有对 Huunyuan 使用那么高。你说的面部和四肢变得扭曲发生在模型的梯度开始因过度训练而崩溃时（例如步数太多，LR 太高）.我个人觉得 2e-4 太多了，我什至没有对 Huunyuan 使用那么高。 你说的面部和四肢变得扭曲发生在模型的梯度开始因过度训练而崩溃时（例如，步数太多，LR 太高）。
> > I will get around to training a character at some point and let you know, but my poor 4070 Ti Super doesn't ever rest lol. I do notice there is a stark lack of character LoRA for Wan I don't know if that's harder or just because I2V is so good or what. Anyway for now this is my most successful Wan training config:我会在某个时候抽身训练一个角色并让你知道，但我可怜的 4070 Ti Super 永远不会休息，哈哈。我确实注意到 Wan 明显缺乏角色 LoRA，我不知道这是更难还是仅仅因为 I2V 太好了或其他什么。无论如何，目前这是我最成功的 Wan 训练配置：我会在某个时候抽身训练一个角色并让你知道，但我可怜的 4070 Ti Super 永远不会休息，哈哈。我确实注意到 Wan 明显缺乏角色 LoRA，我不知道这是更难还是仅仅因为 I2V 太好了或其他什么。无论如何，目前这是我最成功的 Wan 训练配置：我会在某个时候抽身训练一个角色并让你知道，但我可怜的 4070 Ti Super 永远不会休息，哈哈。我确实注意到 Wan 明显缺乏角色 LoRA，我不知道这是更难还是仅仅因为 I2V 太好了或其他什么。无论如何，目前这是我最成功的 Wan 训练配置：
> > Date: 2025-03-14T05:59:03 Title: -------日期：2025-03-14T05：59：03 标题：------- Resolution: 424x240(actual training res) Architecture: wan2.1/lora分辨率：424x240（实际训练分辨率） 架构：wan2.1/lora Network Dim/Rank: 16.0 Alpha: 1.0 dtype: BF16网络亮度/等级：16.0 Alpha：1.0 dtype：BF16 Module: networks.lora_wan : {'loraplus_lr_ratio': '4'}模块： networks.lora_wan ： {'loraplus_lr_ratio'： '4'} Learning Rate (LR): 6e-05学习速率 （LR）：6e-05 Optimizer: came_pytorch.CAME.CAME优化器：came_pytorch。 来。 来 Scheduler: constant_with_warmup Warmup steps: 100计划程序：constant_with_warmup 预热步骤：100 Epoch: 56 Batches per epoch: 40 Gradient accumulation steps: 1Epoch：56 个批次 每个 epoch：40 梯度累积步骤：1 Timestep Sampling: Shift Discrete Flow Shft: 3.0时间步采样：Shift Discrete Flow Shft：3.0日期：2025-03-14T05：59：03 标题：-------日期：2025-03-14T05：59：03 标题：------- 分辨率：424x240（实际训练分辨率） 架构：wan2.1/lora分辨率：424x240（实际训练分辨率） 架构：wan2.1/lora 网络亮度/等级：16.0 Alpha：1.0 dtype： BF16网络亮度/等级：16.0 Alpha：1.0 dtype：BF16 模块： networks.lora_wan ： {'loraplus_lr_ratio'： '4'}模块： networks.lora_wan ： {'loraplus_lr_ratio'： '4'} 学习速率 （LR）：6e-05学习速率 （LR）：6e-05 优化器： came_pytorch.日期： 2025-03-14T05：59：03 蒂特勒： -------日期：2025-03-14T05：59：03 标题：------- 分辨率：424x240（实际训练分辨率） 架构：wan2.1/lora分辨率：424x240（实际训练分辨率） 架构：wan2.1/lora 网络亮度/等级：16.0 Alpha： 1.0 dtype： BF16网络亮度/等级：16.0 Alpha：1.0 dtype：BF16 模块： networks.lora_wan ： {'loraplus_l
> > But my experience training Wan is limited and I'm certain can do better.但我训练 Wan 的经验是有限的，我确信可以做得更好。但我训练 Wan 的经验是有限的，我确信可以做得更好。
> 
> I often see your experience sharing in the AI forum. I am very grateful to you. I use the poorer 4060ti16g, but we can all enjoy the fun of AI. i look forward to your work.我经常在 AI 论坛看到你的经验分享。我非常感谢你。我用的是比较差的 4060ti16g，但我们都可以享受 AI 的乐趣。我期待着你的工作。

Sometimes I will rent a server to train lora. Fortunately, the price of 4090 servers in China is generally about 1.8RMB an hour, 
but The second - hand RTX4090 card，21000RMB(；′⌒`).
